### DISCUSSION TITLE
Project submission button not visible

### DISCUSSION THREAD

**Discussion by Diksha4111** [github/india#883] [Students]
The project submission button is not visible even after updating the Microsoft Azure skill on LinkedIn profile skill page. I mailed the support team several times with the screenshots of the LinkedIn profile skill page and FRT project submission page, but still the button is not visible. I'm waiting for the past 3 days for the button to show up. Today is the last day of submission. 
**Please help me!!!!!!!!!!!!!!**

My LinkedIn profile link: [https://www.linkedin.com/in/diksha-yadav-861bb121a/](url)

LinkedIn profile skill page:
![skills](https://user-images.githubusercontent.com/78994799/184683677-61fb3097-ebd8-41e6-a591-f2028f3047b6.JPG)

FRT portal project submission page:
![frt portal](https://user-images.githubusercontent.com/78994799/184683284-79b49ac6-68b8-4a17-ba84-26a0b7154826.JPG)

**Comment by dikshantchitara** [comment]
same problem what to do now. i have also deadline for project submission. please help us ASAP!

**Reply by mafaraaz** [reply]
Same problem I have invested lot of time and hard work but the submission button not displaying.

**Comment by Diksha4111** [comment]
Project submission deadline is extended till 29th of this month. You can
check, if submit button still not visible just mail them for the same.
